src/pages/ErrorPage.vue

```diff
@@ -1 +1,62 @@
-<template><h1>Error Page</h1></template>
+<template>
+  <div class="container">
+    <h1>404</h1>
+    <h2>Page not found</h2>
+    <h3>The Page you are looking for doesn't exist !</h3>
+    <router-link :to="{ name: 'messages' }"><button>Home</button></router-link>
```

Comment:
You don't need this button tag inside the link, a link is a clickable element
```suggestion
    <router-link :to="{ name: 'messages' }"> Home </router-link>
```